docs/src/developing/intro/rent.md

```diff
@@ -0,0 +1,60 @@
+---
+title: What is rent?
+description: "Rent: the small fee Solana accounts incur to store data on the blockchain. Accounts with >2 years of rent are rent exempt and do not pay the periodic fee."
+keywords: ""
+---
+
+The small fee every Solana Account (or Program) to store data on the blockchain is called "*rent*". This *time and space* based fee is required to keep an account, and its therefore its data, alive on the blockchain since [clusters](../../cluster/overview.md) must actively maintain this data.
```

Comment:
```suggestion
The fee every Solana Account to store data on the blockchain is called "*rent*". This *time and space* based fee is required to keep an account, and its therefore its data, alive on the blockchain since [clusters](../../cluster/overview.md) must actively maintain this data.
```
Imo avoid "small" since this is subjective. Programs are accounts too

Reply:
Subject, maybe. But compared to pretty much every other chain, they are still relatively small. I will remove it still though.
I specifically mentioned programs to help stress that they are lumped into the rent requirements. I will make that more clear.